src/components/inputs/DisplaySelector.tsx

```diff
@@ -0,0 +1,128 @@
+import { BoardViewIcon, DisplayOptionsIcon, ListViewIcon } from '@/icons'
+import { Menu, MenuItem, Stack, styled, Typography } from '@mui/material'
+import { ViewMode } from '@prisma/client'
+import { SecondaryBtn } from '@/components/buttons/SecondaryBtn'
+import { useState } from 'react'
+import { StyledSwitch } from '@/components/inputs/StyledSwitch'
+
+interface Prop {
```

Comment:
Not a big issue but we can name it `DisplaySelectorProps` to keep it consistent with other components